src/Makefile

```diff
@@ -785,6 +802,35 @@ $(SELINUX_POLICY): $(SELINUX_MODULE)
 $(SELINUX_MODULE): $(SELINUX_ENFORCEMENT)
 	checkmodule -M -m -o $@ $?
 
+SYMBOLS_PATH = symbols
+
+ifeq (${TARGET}, winagent)
+strip:
+else
+ifeq (${uname_S}, Linux)
+ifeq (${TARGET}, agent)
+strip: ${BUILD_AGENT} ${BUILD_CMAKE_PROJECTS}
+else
+strip: ${BUILD_SERVER} ${BUILD_CMAKE_PROJECTS}
+endif
+endif
+endif
+
+ifeq (,$(filter ${DISABLE_STRIP_SYMBOLS},YES yes y Y 1))
+	@mkdir -p ${SYMBOLS_PATH}
```

Comment:
Hi @jnasselle 

The symbol folder was created in another PR in the same way as the external folder. It is not necessary to run `mkdir`.